PIM/Integration/02_ManageAttributeSets.md

```diff
@@ -0,0 +1,272 @@
+
+4. Click the [SAVE] button.   
+  The new attribute set has been saved. The *Create Attribute Set* view changes to the *Edit Attribute Set* view. The *Attribute set created* pop-up window is displayed.   
+
+  ![Attribute set created](../../Assets/Screenshots/PIM/Settings/AttributeSets/AttributeSetCreated.png "[Attribute set created]")  
+
+5. Click the ![Back](../../Assets/Icons/Back02.png "[Back]") (Back) button in the upper left corner to return to the list of attribute sets or proceed to [edit the attribute set](#edit-an-attribute-set).
+
+
+### Create an attribute set with attribute copying
+
+When you create an attribute set with attribute copying, you copy all attributes from the selected attribute set once. After copying, the attribute set behaves like an attribute set without attribute inheritance. Further changes in the copied attribute set or not affecting the created attribute set.   
+
+#### Prerequisites
+
+- Additional attributes for the attribute set have been created, see [Create an attribute](./01_ManageAttributes.md#create-an-attribute).
+- The basic data of the attribute set has been completed, see [Define the attribute set basic data](#define-the-attribute-set-basic-data)
+
+#### Procedure
+*PIM > Settings > Tab ATTRIBUTE SETS > Button Add*
+
+![Create attribute set inheritance](../../Assets/Screenshots/PIM/Settings/AttributeSets/CreateAttributeSetInheritance.png "[Create attribute set inheritance]")
+
+1. Select an attribute set in the *Inherit/Copy Values from* drop-down list.
+
+2. Enable the *Inherit Configuration* toggle if you also want to adopt the configuration to the new attribute set.
+
+3. Select the **Copy once** option in the *Inherit/Copy Values* drop-down list.
+
+4. Click the [SAVE] button.   
+  The new attribute set has been saved. The *Create Attribute Set* view changes to the *Edit Attribute Set* view. The *Attribute set created* pop-up window is displayed.   
+
+  ![Attribute set created](../../Assets/Screenshots/PIM/Settings/AttributeSets/AttributeSetCreated.png "[Attribute set created]")  
+
+5. Click the ![Back](../../Assets/Icons/Back02.png "[Back]") (Back) button in the upper left corner to return to the list of attribute sets or proceed to [edit the attribute set](#edit-an-attribute-set).
+
+
+
+## Edit an attribute set
+
+After you have created an attribute set, you can edit it. Depending on the selected inheritance type, only a certain number of attributes may be editable. However, you can add further attributes to an attribute set or remove unmapped attributes from an attribute set.
+
+### Add an attribute to the set
+
+Add a new attribute to an attribute set to define any further specifications.
+
+#### Prerequisites
+
+- An attribute set has been created, see [Create an attribute set](#create-an-attribute-set).
+- At least one additional attribute has been created, see [Create an attribute](./01_ManageAttributes.md#create-an-attribute).
```

Comment:
```suggestion
- At least one additional attribute has been created, see [Create an attribute](./01_ManageAttributes.md#create-an-attribute).
```

Reply:
Wegglassen? Ist das nicht in den Prerequisites von Create an attribute set enthalten?